BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -114,22 +116,31 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
             // Excludes files with .x64 extension to avoid flagging Professional files.
             // Sorts list of files in ZDAT* so just the first file gets pulled, later ones have a chance of the ring 
             // intersecting the start of the file.
-            var fileList = files.Where(f => !f.EndsWith(".x64", StringComparison.OrdinalIgnoreCase))
-                .Where(f =>
+
+            string[] dirs = ["ZDAT", "ZDATA", "System"];// Kenny's Adventure uses System instead of ZDAT.
+            List<string>? lightFiles = null;
+            // TODO: Compensate for the check being run a directory or more higher
+            var fileList = files.Where(f => !f.EndsWith(".x64", StringComparison.OrdinalIgnoreCase));
+            {
+                foreach (var dir in dirs)
                 {
-                    // TODO: Compensate for the check being run a directory or more higher
-                    f = f.Remove(0, path.Length);
-                    f = f.TrimStart('/', '\\');
-                    return f.StartsWith("ZDAT", StringComparison.OrdinalIgnoreCase);
-                })
-                .OrderBy(f => f)
-                .ToList();
-
-            if (fileList.Count > 0)
+                    lightFiles = fileList.Where(f =>
+                        {
+                            f = f.Remove(0, path.Length);
+                            f = f.TrimStart('/', '\\');
+                            return f.StartsWith(dir + Path.DirectorySeparatorChar, StringComparison.OrdinalIgnoreCase);
+                        })
+                        .OrderBy(f => f)
+                        .ToList();
+                    if (lightFiles.Count() > 0)
+                        break;
+                }
+            }
+            if ((lightFiles != null) && (lightFiles.Count > 0))
```

Comment:
Once the above block is de-indented, please add a newline above this `if` statement.